### PR TITLE
tikv: rename auto-tuned config 

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)
Rename `auto-tuned` to `rate-limiter-auto-tuned` to avoid confusion. Also update default value related to auto-tuned.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/4422
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -733,22 +733,22 @@ rocksdb 相关的配置项。
...
 + 默认值：2
 + 最小值：1
 + 最大值：3
 
-### `auto-tuned`
+### `rate-limiter-auto-tuned`
 
-+ 开启自动优化 Rate LImiter 的配置的开关。
-+ 默认值：false
++ 开启自动优化 RocksDB 的 compaction rate limiter 配置的开关。
```
Comment (yiwu-arbug) [inline tikv-configuration-file.md:750]:
```suggestion
+ 开启自动优化 RocksDB 的 compaction rate limiter。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -733,22 +733,22 @@ rocksdb 相关的配置项。
...
++ RocksDB compaction rate limiter 的限制速率。
++ 默认值：10GB
 + 最小值：0
-+ 单位：Bytes
++ 单位：B|KB|MB|GB
 
 ### `rate-limiter-mode`
 
-+ Rate LImiter 模式，取值：1（ReadOnly），2（WriteOnly），3（AllIo）。
++ RocksDB 的 compaction rate limiter 模式，取值：1（ReadOnly），2（WriteOnly），3（AllIo）。
```
Comment (TomShawn) [inline tikv-configuration-file.md:743]:
```suggestion
+ RocksDB 的 compaction rate limiter 模式
+ 可选值：1 (ReadOnly)，2 (WriteOnly)，3 (AllIo)
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -733,22 +733,23 @@ rocksdb 相关的配置项。
...
++ RocksDB compaction rate limiter 的限制速率。
++ 默认值：10GB
 + 最小值：0
-+ 单位：Bytes
++ 单位：B|KB|MB|GB
 
 ### `rate-limiter-mode`
 
-+ Rate LImiter 模式，取值：1（ReadOnly），2（WriteOnly），3（AllIo）。
++ RocksDB 的 compaction rate limiter 模式
```
Comment (TomShawn) [inline tikv-configuration-file.md:743]:
```suggestion
+ RocksDB 的 compaction rate limiter 模式。
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -733,22 +733,23 @@ rocksdb 相关的配置项。
...
 + 默认值：2
 + 最小值：1
 + 最大值：3
 
-### `auto-tuned`
+### `rate-limiter-auto-tuned`
 
-+ 开启自动优化 Rate LImiter 的配置的开关。
-+ 默认值：false
++ 开启自动优化 RocksDB 的 compaction rate limiter。
```
Comment (TomShawn) [inline tikv-configuration-file.md:751]:
```suggestion
+ 开启自动优化 RocksDB 的 compaction rate limiter 配置。
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Pull request merged**